static/sass/elements/chromedash-metrics.scss

```diff
@@ -57,14 +57,19 @@
         }
       }
 
-      x-meter {
+      chromedash-x-meter {
```

Comment:
The only Sass in here is the nesting and variable reference below. If you manually expanded the nested and made the variable a CSS variable, this file could be inlined into `chromedash-metrics.js`